tests/components/mqtt/test_device_trigger.py

```diff
@@ -70,7 +70,6 @@ async def test_get_triggers(
             "platform": "device",
             "domain": DOMAIN,
             "device_id": device_entry.id,
-            "discovery_id": "bla",
```

Comment:
`discovery_id` is no longer exposed, so it is no longer added to the device automation trigger config by default.